src/offline/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from 'react';
+
+import { getOfflines, getRemainingOfflines } from 'frontpage/api/offline';
+import { IOfflineIssue } from 'frontpage/models/Offline';
+import OfflineList from './components/OfflineList';
+import Spinner from 'common/components/Spinner';
+
+export const Offline: React.FC = () => {
+  const [offlines, setOfflines] = useState<IOfflineIssue[]>([]);
+
+  /** Get the first batch of Offlines for feast loading */
```

Comment:
Loading is a feast 🥳 🎈

Reply:
Hahaha, I shamelessly copied this file from frontpage for faster development, but it should probably be refactored to share as much as possible with frontpage. Noticed the comment there, but forgot to fix it so now it is present in two files 😂

Reply:
Yeah, I had i feeling I was the original sinner in this case 🙈